changes/510.changed

```diff
@@ -0,0 +1,3 @@
+Removed dependency on `django-mptt`, migrating `Region`, `RackGroup`, `TenantGroup`, and `InventoryItem` to use `django-tree-queries` instead. This does change the API for certain tree operations on these models, for example `get_ancestors()` is now `ancestors()` and `get_descendants()` is now `descendants()`.
```

Comment:
Should this not partially be `changes/510.removed`?

Reply:
Probably so. Good thought.